### ISSUE TITLE
Homebrew-science / Homebrew / Linuxbrew

### ISSUE THREAD

**Issue by sjackman** [ropensci/unconf#34]
I'm the maintainer of [Linuxbrew](http://brew.sh/linuxbrew/), a maintainer of [Homebrew-science](http://brew.sh/homebrew-science/), and a contributor to [Homebrew](http://brew.sh/). I wanted to post my involvement with these projects to say that I'm happy to make any improvements that could make your R hacking more effective.

Cheers,
Shaun

**Comment by gaborcsardi** [comment]
Related: I think it would be great to have some mappings from the `SystemRequirements` field in `DESCRIPTION` to the names of the corresponding libraries/tools on various platforms. Then the installer could give a more meaningful error message. So we could have something like this for Homebrew, Debian, Windows, etc. 

So how about creating a `sysreqmaps` package?

**Comment by sjackman** [comment]
Yes. @eeddelbuettel and I have been discussing this matter in #25.

**Comment by sjackman** [comment]
I've created a very thin R package over the Homebrew command line client `brew`.
https://github.com/sjackman/homebrewr

``` r
brew_install("hello")
brew_remove("hello")
brew_update()
brew_upgrade()
```

**Comment by sjackman** [comment]
```r
brew_install("hello")
brew_remove("hello")
brew_update()
brew_upgrade()
```
